services/galley/src/Galley/API/MLS/Message.hs

```diff
@@ -532,6 +543,12 @@ postMLSMessageToRemoteConv loc qusr _senderClient con smsg rcnv = do
     MLSMessageResponseProtocolError e ->
       throw (mlsProtocolError e)
     MLSMessageResponseProposalFailure e -> throw (MLSProposalFailure e)
+    MLSMessageResponseUnreachableBackends ds ->
+      throw . InternalErrorWithDescription $
```

Comment:
de-duplicate this particular internal error and give it a name?

Reply:
... as in to have a new data constructor for `InternalError` that is suited for this specific purpose?